assessments/0010-etcd/etcd-issues.md

```diff
@@ -46,7 +47,7 @@ Rewrite each Tutorial as step-by-step instructions for a single task in an indiv
 
 ## Issue: Write installation instructions for Kubernetes
 
-Write the procedure [Installation as part of Kubernetes installation](install/#installation-as-part-of-kubernetes-installation).
+Write the procedure [Installation as part of Kubernetes installation](install/#installation-as-part-of-kubernetes-installation), or link to Kubernetes documentation that includes etcd installation as backstore. Explain and fill in (on the etcd docs page) any gaps in the procedure.
```

Comment:
@dwelsch-esi - these link starting with `install/...` are invalid. What were they mean to refer to? /cc @nate-double-u

Reply:
Ah, nm, found it: the `https://etcd.io/docs/v3.5/` prefix was missing.